npm/react/cypress/plugins/index.js

```diff
@@ -55,15 +54,11 @@ const webpackOptions = {
   },
 }
 
-const options = {
-  // send in the options from your webpack.config.js, so it works the same
-  // as your app's code
-  webpackOptions,
-  watchOptions: {},
-}
-
-module.exports = (on, config) => {
-  on('file:preprocessor', webpackPreprocessor(options))
+const cypressPluginsFn = (on, config) => {
```

Comment:
For me this looks a bit confusing in comparison with other cypress examples. Maybe we can do it simply:

```suggestion
module.exports = (on, config) => {
```